### PR TITLE
[TASK] Remove outdated menuitems

### PR DESCRIPTION
**Pull request by linawolf**
move generalitemstates and general menu properties to TMENU, as that is the only surviving menu

### CODE REVIEW

**Comment by brotkrueml** [comment]
It is intended hat TMENU.sectionIndex is a separate page and not included into the TMENU page?

File: Documentation/MenuObjects/Tmenu/Index.rst [hunk 1]
```diff
@@ -9,11 +9,235 @@ TMENU
...
+
+   Description
+         (See above)
+
+
+.. ###### END~OF~TABLE ######
+
+[tsref:(cObject).HMENU.(mObj\_itemStates)]
+
+Order of priority: USERDEF2, USERDEF1, SPC, USR, CURIFSUB, CUR,
```
Comment (brotkrueml) [inline Documentation/MenuObjects/Tmenu/Index.rst:224]: I would like to see this as a note, so it is more prominent (either here or at the beginning of the item states list).

**Comment by linawolf** [comment]
> It is intended hat TMENU.sectionIndex is a separate page and not included into the TMENU page?

Right, that doesn't really make sense anymore, I will move it

File: Documentation/MenuObjects/Tmenu/Index.rst [hunk 2]
```diff
@@ -125,24 +360,342 @@ TMENU
...
+.. container:: table-row
+
+   Property
+         sectionIndex
+
+   Data type
+         *(array of keys)*
+
+   Description
+         (see below)
```
Comment (brotkrueml) [inline Documentation/MenuObjects/Tmenu/Index.rst:395]: "see below" is not correct as it is by now a separate page. So either include it into this page or link it accordingly to the other page.

**Comment by brotkrueml** [comment]
Apart from the comments, it is okay for me. In a second PR we can format some stuff accordingly (like HTML as HTML and so on). Can take care of this after merging of this PR.

File: Documentation/MenuObjects/Tmenu/Index.rst [hunk 3]
```diff
@@ -9,11 +9,235 @@ TMENU
 
 .. note::
 
-   TMENU is a menu object type.
+   :typoscript:`TMENU` is a menu object type. In TYPO3 10 the remaining other menu types
```
Comment (brotkrueml) [inline Documentation/MenuObjects/Tmenu/Index.rst:12]:
```suggestion
   :typoscript:`TMENU` is a menu object type. In TYPO3 v10 the remaining other menu types
```

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**